app/views/ad_templates/default/show.html.erb

```diff
@@ -5,8 +5,7 @@
         <a data-href="campaign_url" class="cf-img-wrapper" target="_blank" rel="noopener">
           <%= image_tag url_for(@campaign.creative.large_image), border: 0, height: 100, width: 130, class: "cf-img" %>
         </a>
-        <a data-href="campaign_url" class="cf-text" target="_blank" rel="noopener">
-          <strong><%= @campaign.creative.headline %></strong>
+        <a data-href="campaign_url" class="cf-text" target="_blank" rel="noopener"> <strong><%= @campaign.creative.headline %></strong>
```

Comment:
Any reason to pull this up onto a single line?

Reply:
Weird, I'll fix it